nexteuropa_formatters/nexteuropa_formatters.module

```diff
@@ -681,13 +681,16 @@ function _nexteuropa_formatters_file_markup($file, $modifier = NULL, $subfile =
     $file_info_string = isset($file_language_string) ? '<span class="file__content-language">' . $file_language_string . ' </span>' : '';
     $file_info_string .= !empty($file_info_parts) ? '<span class="file__properties">(' . implode(' - ', $file_info_parts) . '</span>)' : '';
 
-    // Use the description as the link text if available.
-    if (isset($file->entity)) {
-      $file_node = entity_metadata_wrapper('node', $file->entity);
-      $title_string = $file_node->title->value();
-    }
-    else {
-      $title_string = !empty($file->description) ? $file->description : $file->filename;
+    // DTTSB-2132 - Commissioners site: mission letter not displaying title.
+    if (empty($file->filename_field)) {
```

Comment:
Basically what i've found is that, apparently only in the commissioners, we are using the filename_field to replace the title of the file entity. The logic i changed was always overriding the file title, basically, using the title or the description of the referencing entity, the ticket is about files that were indeed showing the title of the wrapping entity, this seems to be the solution then, when a title has been explicitly set through the filename_field we don't override it.